src/darwin.rs

```diff
@@ -1,24 +1,18 @@
-use mach::task::task_info;
-use mach::task_info::{
-    task_basic_info, task_basic_info_t, task_thread_times_info, task_thread_times_info_t, MIG_ARRAY_TOO_LARGE,
-    TASK_BASIC_INFO, TASK_BASIC_INFO_COUNT, TASK_THREAD_TIMES_INFO, TASK_THREAD_TIMES_INFO_COUNT,
+use libc::{
+    integer_t, kern_return_t, mach_task_basic_info, mach_task_self, rusage, task_thread_times_info, time_value_t,
+    timespec, timeval, EFAULT, EINVAL, EPERM, KERN_INVALID_ARGUMENT, KERN_SUCCESS, MACH_TASK_BASIC_INFO,
+    MACH_TASK_BASIC_INFO_COUNT, RUSAGE_CHILDREN, RUSAGE_SELF, TASK_THREAD_TIMES_INFO, TASK_THREAD_TIMES_INFO_COUNT,
 };
+use std::ffi::CStr;
+use std::mem::MaybeUninit;
+use std::time::Duration;
 
-use mach::time_value::{time_value_add, time_value_t};
-
-use mach::traps::mach_task_self;
-
-use mach::kern_return::{KERN_INVALID_ARGUMENT, KERN_SUCCESS};
-
-use libc;
-use libc::rusage;
-use libc::timespec;
-use libc::timeval;
-use libc::{EFAULT, EINVAL, EPERM, RUSAGE_CHILDREN, RUSAGE_SELF};
+pub const MIG_ARRAY_TOO_LARGE: kern_return_t = -307;
 
 use super::*;
 
 use utils::CpuTime;
+const VOLTAGE_STATE5_SRAM: &[u8] = b"voltage-states5-sram";
```

Comment:
unused if not `#[cfg(all(target_os = "macos", target_arch = "aarch64"))]`?

L6 `use std::ffi::CStr;` same.